{{ cookiecutter.repo_name }}/buildSrc/src/main/java/dependencies/dependencies.kt

```diff
@@ -25,9 +25,10 @@ object Versions {
     const val kotlin = "1.3.31"
     const val ktlint = "0.33.0"
     const val okhttp = "3.14.2"
-    const val retrofit = "2.5.0"
+    const val retrofit = "2.6.0"
     const val roboletric = "4.3"
     const val room = "2.1.0-rc01"
+    const val coroutines = "1.1.1"
```

Comment:
I believe the latest release is [version 1.3.0-RC2](https://github.com/Kotlin/kotlinx.coroutines/releases/tag/1.3.0-rc2)